services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/settings.py

```diff
@@ -338,6 +340,9 @@ def _merge_resources_in_settings(
                     "MemoryBytes"
                 ] += resource_value.reservation
             else:  # generic resources
+                if skip_generic_resources:
+                    # NOTE: placement constraints will be used in favour of generic resources
```

Comment:
so the proposal would be to go through the generic resources and keep the ones that are **not** in the DIRECTOR_V2_PLACEMENT_CONSTRAINTS_REPLACEMENTS_FOR_GENERIC_RESOURCES variable